ssv/pre_consensus_justification.go

```diff
@@ -0,0 +1,129 @@
+package ssv
+
+import (
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/pkg/errors"
+)
+
+func (b *BaseRunner) shouldProcessingJustificationsForHeight(msg *qbft.SignedMessage) bool {
+	rightQBFTHeight := b.QBFTController.CanStartInstance() == nil && b.QBFTController.Height+1 == msg.Message.Height
+	hasData := len(msg.FullData) > 0
+	return rightQBFTHeight && hasData
+}
+
+func (b *BaseRunner) validatePreConsensusJustifications(data *types.ConsensusData) error {
+	if err := data.Validate(); err != nil {
+		return err
+	}
```

Comment:
in data.Validate, or somewhere else, do we check that the data contains PreConsensusJustifications at all?

Reply:
inside consensusData.validate()